.travis.yml

```diff
@@ -1,13 +1,25 @@
-language: node_js
-nodejs:
-  - 9
+matrix:
+  include:
+    - language: node_js
+      nodejs:
+        - 9
 
-addons:
-  apt:
-    packages:
-      - python-pygments
+      addons:
+        apt:
+          packages:
+            - python-pygments
 
-before_install:
-  - export PATH="$TRAVIS_BUILD_DIR/bin:$PATH"
+      before_install:
+        - export PATH="$TRAVIS_BUILD_DIR/bin:$PATH"
```

Comment:
Are these spacing changes intentional?

Reply:
Yes. YAML depends on spacing to dictate what an object lives under. So as I added a second language here, I needed to move these over they would be under `matrix` and `include`. 

The spacing that doesn't matter are the new lines between objects, but I just followed what was here to be consistent.

Reply:
Perfect. Thanks!